common/src/main/java/org/dash/wallet/common/ui/dialogs/AdaptiveDialog.kt

```diff
@@ -44,6 +44,7 @@ class AdaptiveDialog(@LayoutRes private val layout: Int): DialogFragment() {
         private const val MESSAGE_ARG = "message"
         private const val POS_BUTTON_ARG = "positive_text"
         private const val NEG_BUTTON_ARG = "negative_text"
+        private const val EXTRA_MESSAGE_BUTTON_ARG = "extra_message_text"
```

Comment:
I don't think we should be amending AdaptiveDialog for every possible case. This is a simple class for a simple purpose, if we start adding more stuff to it it'll bloat. 

If you require more complex behavior like additional buttons, you can create another dialog (make sure it's DialogFragment), or even try to derive from this one.

Reply:
I think this is a good point.  If we change the interface of `AdaptiveDialog`, then we should have QA test all places where `AdaptiveDialog` is used in the app, or at least all the ones affected by the change.  It may be difficult to identify all of those places.

If a different dialog class is used, then that will limit the places that are affected by the change.  We may find that the new class is useful in other places.  There are still many old style dialogs in our app that will need to be replaced eventually.